_sass/components/_card.scss

```diff
@@ -79,3 +79,33 @@
     line-height: 32px;
   }
 }
+
+.sponsor-card {
+  width: 268px;
+  .sponsor-card-image {
+    width: 268px;
+    height: 268px;
+    object-fit: contain;
+  }
+  .sponsor-card-title {
+    color: $primary;
+    font-weight: 700;
+    font-size: 20px;
+    line-height: 32px;
+  }
+  .sponsor-card-content {
+    width: 268px;
+    height: 130px;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 26px;
+  }
```

Comment:
When the browser is narrow, things start to look weird:

<img width="709" alt="image" src="https://user-images.githubusercontent.com/594058/206870342-d648e9c4-c4e7-4dda-aa18-b5942d6d4a5c.png">

Please reconsider the responsive behavior.